git/homework.py

```diff
@@ -125,9 +150,16 @@ def alphabet() -> dict:
     Notes You could see an implementaion of this one in test, but create another one
     Examples:
         alphabet()
-        >>> {"a": 1, "b": 2 ...}
+    >>> {"a": 1, "b": 2 ...}
     """
-    pass
+    i = 1
+    my_alphabet = {}
+    while i <= len(string.ascii_lowercase):
+        for letter in string.ascii_lowercase:
+            my_alphabet[letter] = i
+            i += 1
+
+    print(my_alphabet)
```

Comment:
do not print data, you need to return
also you can simplify this func

Reply:
I know, that I may use the dict comprehension, but I don`t know, how to do this. So I did it how I already knew.